tests/scf/test_x2c1e.py

```diff
@@ -90,6 +90,7 @@ def test_sox2c1e_water():
         basis_set="decon-cc-pvdz",
         auxiliary_basis_set="cc-pvtz-jkfit",
         x2c_type="so",
+        snso_type=None,
```

Comment:
Make sure that the documentation of this option is updated since `None` is not listed in the current docs:

> The type of screened nuclear spin-orbit coupling scaling scheme to use. Only relevant if x2c_type is “so”. Options are “boettger”, “dc”, “dcb”, or “row-dependent”

Please indicate the default value in this description and add a period at the end.